packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/notes-summary.component.tsx

```diff
@@ -12,21 +12,21 @@ interface NotesSummaryProps {
 const NotesSummary: React.FC<NotesSummaryProps> = ({ notes }) => {
   const { t } = useTranslation();
 
+  if (notes.length === 0) {
+    return <EmptyState displayText={t('notes__lower', 'notes')} headerTitle={t('notes', 'Notes')} />;
+  }
```

Comment:
I've moved the empty state rendering logic up here for an early return. This should make the logic simpler to reason about.